homeassistant/config_entries.py

```diff
@@ -1623,7 +1623,7 @@ async def async_step_zeroconf(
     def async_create_entry(
         self,
         *,
-        title: str,
```

Comment:
The PR is now adjusted so `ConfigEntry.title` is always a str. We should perhaps still adjust the prints in case the config entry has no title, in the case above, perhaps:

```py
            _LOGGER.warning(
                "Config entry '%s' for %s integration %s",
                self.title or "<untitled>",
                self.domain,
                auth_message,
            )
```

Reply:
Sounds good 👍